### PR TITLE
chore: Add DpCheckboxGroup to Storybook.

### PR DESCRIPTION
**Pull request by ahmad-demos**
- Add DpCheckboxGroup to Storybook.

### CODE REVIEW

File: src/components/DpCheckboxGroup/DpCheckboxGroup.vue [hunk 1]
```diff
@@ -22,6 +22,7 @@
 <script>
 import { CleanHtml } from '../../directives'
 import DpCheckbox from '../DpCheckbox/DpCheckbox'
+import Vue from "vue";
```
Comment (salisdemos) [inline src/components/DpCheckboxGroup/DpCheckboxGroup.vue:25]: we don't want to import vue here. Vue has to be accessible globally.

**Review by salisdemos — CHANGES_REQUESTED** [review]

File: src/components/DpCheckboxGroup/DpCheckboxGroup.vue (hunk 1 above)
Comment (ahmad-demos) [reply src/components/DpCheckboxGroup/DpCheckboxGroup.vue:25]: Yes but I have this error message: `ReferenceError: "Vue is not defined".`

Comment (salisdemos) [reply src/components/DpCheckboxGroup/DpCheckboxGroup.vue:25]: I think what You actually want to do is something like https://github.com/demos-europe/demosplan-ui/pull/180/files

Comment (salisdemos) [reply src/components/DpCheckboxGroup/DpCheckboxGroup.vue:25]: or approve the PR and merge the changes into Your PR ;-)

**Review by salisdemos — CHANGES_REQUESTED** [review]

File: src/components/DpCheckboxGroup/DpCheckboxGroup.vue (hunk 1 above)
Comment (ahmad-demos) [reply src/components/DpCheckboxGroup/DpCheckboxGroup.vue:25]: approved

Comment (ahmad-demos) [reply src/components/DpCheckboxGroup/DpCheckboxGroup.vue:25]: Merge done!

Comment (ahmad-demos) [reply src/components/DpCheckboxGroup/DpCheckboxGroup.vue]: Done in https://github.com/demos-europe/demosplan-ui/pull/477/commits/92c9efaa1ffdb39447a262752008ec8c3378cba7

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 
 ### Added
 
+- ([#477](https://github.com/demos-europe/demosplan-ui/pull/477)) Add DpCheckboxGroup documentation to Storybook7 ([@ahmad-demos](https://github.com/@ahmad-demos))
 - ([#472](https://github.com/demos-europe/demosplan-ui/pull/472)) Add DpCard documentation to Storybook7 ([@ahmad-demos](https://github.com/@ahmad-demos))
```
Comment (salisdemos) [inline CHANGELOG.md:10]: I think we should merge these two entries...

Comment (ahmad-demos) [reply CHANGELOG.md]: Done in https://github.com/demos-europe/demosplan-ui/pull/477/commits/e024e70f20f273bf4cedb3a28ba4ad05abde26c9

**Review by salisdemos — APPROVED** [review]

**Pull request merged**